coordinator/src/node.rs

```diff
@@ -49,31 +48,29 @@ pub struct Node {
 }
 
 impl Node {
-    /// Returns true or false, whether we can find an useable channel with the provided trader.
+    /// Returns true or false, whether we can find an usable channel with the provided trader.
     ///
-    /// Note, we use the useable channel to implicitely check if the user is connected, as it
-    /// wouldn't be useable otherwise.
+    /// Note, we use the usable channel to implicitely check if the user is connected, as it
```

Comment:
Actually both ways of writing it are correct. usable and useable. 😜 

https://english.stackexchange.com/questions/69709/is-useable-preferred-in-certain-regions-or-just-an-alternate-spelling-of-usa